feature/authentication/src/main/java/com/finance2up/authentication/presentation/ui/otp/OTPScreen.kt

```diff
@@ -0,0 +1,217 @@
+import androidx.compose.ui.draw.alpha
+import androidx.compose.ui.focus.FocusRequester
+import androidx.compose.ui.focus.focusRequester
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.painter.Painter
+import androidx.compose.ui.res.dimensionResource
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.input.ImeAction
+import androidx.compose.ui.text.input.KeyboardType
+import androidx.hilt.navigation.compose.hiltViewModel
+import androidx.lifecycle.compose.collectAsStateWithLifecycle
+import androidx.navigation.NavController
+import com.finance2up.authentication.R
+import com.finance2up.authentication.presentation.util.fontSizeDimensionResource
+
+@Composable
+fun OTPScreen(navController: NavController, emailUser: String?) {
+    val otpViewModel: OTPViewModel = hiltViewModel()
+
+    val firstText = otpViewModel.firstText.collectAsStateWithLifecycle()
+    val secondText = otpViewModel.secondText.collectAsStateWithLifecycle()
+    val thirdText = otpViewModel.thirdText.collectAsStateWithLifecycle()
+    val forthText = otpViewModel.forthText.collectAsStateWithLifecycle()
+    val oTPUIState = otpViewModel.oTPUIState.collectAsStateWithLifecycle()
+    Column(
+        modifier = Modifier
+            .padding(dimensionResource(id = R.dimen.padding_otp_parentView))
+            .fillMaxSize()
+    ) {
+        Column(
+            modifier = Modifier.fillMaxWidth(),
+        ) {
+
+            Text(
+                stringResource(R.string.otp_title), style = MaterialTheme.typography.h5.copy(
+                    fontWeight = FontWeight.Bold,
+                    fontSize = fontSizeDimensionResource(id = R.dimen.textSize_otp_title)
+                )
+            )
+            Text(
+                stringResource(R.string.otp_checkEmail),
+                style = MaterialTheme.typography.h5.copy(
+                    fontSize = fontSizeDimensionResource(id = R.dimen.textSize_otp_checkEmail)
+                ),
+                modifier = Modifier.padding(top = dimensionResource(id = R.dimen.paddingTop_otp_checkEmail))
+            )
+
+            val painter: Painter = painterResource(id = R.drawable.logo_otp)
+            Image(
+                painter = painter, contentDescription = "My Image", modifier = Modifier
+                    .width(
+                        dimensionResource(id = R.dimen.width_otp_imageOTP)
+                    )
+                    .height(
+                        dimensionResource(id = R.dimen.height_otp_imageOTP)
+                    )
+                    .padding(
+                        top = dimensionResource(id = R.dimen.paddingTop_otp_imageOTP)
+                    )
+                    .align(Alignment.CenterHorizontally)
+            )
+
+            Text(
+                text = "($emailUser)",
```

Comment:
What's this? cú pháp này lạ quá

Reply:
nhận dữ liệu từ màn PreOTPScreen và c hiển thị ở Text 
![image](https://user-images.githubusercontent.com/84829229/227458106-a0ef7d45-885a-4a78-aa69-59bb4f0f3a93.png)